querybook/webapp/hooks/dag/useGraphQueryCells.ts

```diff
@@ -0,0 +1,65 @@
+import { queryCellDraggableType } from 'components/DataDocDAGExporter/DataDocDAGExporter';
+import { IDataQueryCell } from 'const/datadoc';
+import { useShallowSelector } from 'hooks/redux/useShallowSelector';
+import React from 'react';
+import { useDrop } from 'react-dnd';
+
+import { IStoreState } from 'redux/store/types';
+import * as dataDocSelectors from 'redux/dataDoc/selector';
+
+import { IDragItem } from 'ui/DraggableList/types';
+
+export function useGraphQueryCells(docId, savedNodes) {
+    const { dataDocCells } = useShallowSelector((state: IStoreState) =>
```

Comment:
useSelector here, use useShallowSelect for cases when you want to return an object, like:

```
useShalldowSelector(state => ({ docs: state.dataDocs, tables: state.dataSource.tables }))